.travis.yml

```
@@ -15,11 +20,11 @@ services:
 - mongodb
 - redis-server
 - cassandra
+before_install:
+- phpenv config-rm xdebug.ini || return 0
```

HHVM does not have this file. Just silently fail ig XDebug is not available